package.json

```diff
@@ -15,11 +16,15 @@
     "@fortawesome/fontawesome-svg-core": "^6.5.2",
     "@fortawesome/free-solid-svg-icons": "^6.5.2",
     "@fortawesome/react-fontawesome": "^0.2.0",
+    "@types/cookie": "^0.6.0",
```

Comment:
You can probably move these `@types/*` to your devDependencies

Reply:
Thanks Brian. I will make that change.